### PR TITLE
added RSA-key-pair generator

### PR DESCRIPTION
**Pull request by obiwan69**

### CODE REVIEW

**Comment by obiwan69** [comment]
added the requirements.txt file

**Comment by Kaushl2208** [comment]
Hey @obiwan69 , good work. One last thing, Can you rebase these commits? You can refer to this [Rebasing](https://help.github.com/en/articles/about-git-rebase)

**Comment by obiwan69** [comment]
Rebased :)

File: RSA-key-pairs/requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+All requirements already present in Python packages
```
Comment (hastagAB) [inline RSA-key-pairs/requirements.txt]: Please remove this line as this is not required.

**Comment by obiwan69** [comment]
Removed redundancy on requirement.txt and added script to README.md . Squashed to one commmit.

**Pull request merged**